python/VIRTUAL-README.md

```diff
@@ -0,0 +1,57 @@
+# Python Virtual Environment
+
+It is recommended that Python development be done in an isolated environment called a virtual environment.  There are multiple ways to set up and use Python virtual environments.  This README describes one of those ways.
+
+The basic steps are:
+1. Install virtualenv (done once)
+2. Create a virtual environment (done once per development effort)
+3. Repeat as needed:
+    1. Activate a virtual environment
+    2. Install libraries and develop code
+    3. Deactivate a virtual environment
+
+Follow the instructions below to set up your virtual environment. It is important to note that you should never install any project specific python dependencies outside of your virtual environment. Also, ensure that your virtual environment has been activated before running python scripts within this repository.
+
+## Install virtualenv
+
+If you haven't installed `virtualenv` yet, follow these steps. This only needs to be once.
+
+```bash
+$ cd ~
+$ mkdir .virtualenvs
+$ pip install virtualenv
+```
+Verify virtualenv is installed
+```bash
+$ which virtualenv
+```
+
+## Create virtual environment
+
+When you are starting development of a python project, you first need to create a virtual environment.  The name of the virtual environent in the example below, `aida-interchange-format`, assumes you are making changes or testing the AIF library.  Feel free to use a name specific to your application if you are just using the AIF library.
+
+Navigate inside the `AIDA-Interchange-Format/python` directory and run the following
+
+```bash
+$ virtualenv -p python3 ~/.virtualenvs/aida-interchange-format
```

Comment:
FWIW, I believe the `-p python3` is only needed if the default `python` command isn't version 3.x (presumably, v2.7.x).  On my Windows machine, python is python3, so including the `-p` results in an error.  I don't think this needs to be mentioned here, because if you're using Python on Windows, you probably know whether or not you need the `-p python3`.

Reply:
If we get feedback from teams that they are using Windows and this is confusing, we can update documentation throughout the repo.